crates/store/re_chunk/src/batcher.rs

```diff
@@ -784,18 +784,35 @@ pub struct PendingRow {
     /// The component data.
     ///
     /// Each array is a single component, i.e. _not_ a list array.
-    pub components: IntMap<ComponentDescriptor, ArrayRef>,
+    pub components: IntMap<ComponentIdentifier, (ComponentDescriptor, ArrayRef)>,
```

Comment:
This then becomes a `SerializedComponentBatch`, I believe.
```suggestion
    pub components: IntMap<ComponentIdentifier, SerializedComponentBatch>,
```